requirements.txt

```diff
@@ -0,0 +1,104 @@
+actionlib==1.12.1
```

Comment:
Please remove all the ROS specific stuff from here and include only what is required for the imports in the files you created. You can add to the README that ROS version XYZ is required